Sources/SwifterSwift/Foundation/URLExtensions.swift

```diff
@@ -101,6 +107,27 @@ public extension URL {
         }
     }
 
+    /// SwifterSwift: Returns a new URL by removing all the query parameters.
+    ///
+    ///     let url = URL(string: "https://google.com?code=12345")!
+    ///     print(url.deletingAllQueryParameters()) // prints "https://google.com"
+    ///
+    /// - Returns: URL with all query parameters removed.
+    func deletingAllQueryParameters() -> URL {
+        var urlComponents = URLComponents(url: self, resolvingAgainstBaseURL: true)!
+        urlComponents.queryItems = nil
+        return urlComponents.url!
+    }
+    
+	/// SwifterSwift: Remove all the query parameters from the URL.
+	///
+	///        var url = URL(string: "https://google.com?code=12345")!
+	///        url.deleteAllQueryParameters()
+	///        print(url) // prints "https://google.com"
+    mutating func deleteAllQueryParameters() {
+        self = deletingAllQueryParameters(parameters)
```

Comment:
One of the conditions of SwifterSwift is that you can't use call our own functions, they should all be easily copy-pasted individually. It's not the most efficient way, but it suits how most people use this library.
```suggestion
        var urlComponents = URLComponents(url: self, resolvingAgainstBaseURL: true)!
        urlComponents.queryItems = nil
        self = urlComponents.url!
```